.github/workflows/audit.yml

```diff
@@ -0,0 +1,18 @@
+name: Audit
+
+on:
+  schedule:
+    - cron: '0 0 * * *'
+  push:
+    paths:
+      - "**/Cargo.lock"
+      - "**/Cargo.toml"
+
+jobs:
+  audit:
```

Comment:
```suggestion
  audit-rust:
```
It might be good to make the job title a bit more descriptive

Reply:
Yeah, and we also should look for npm audit options.

Reply:
I personally recommend [Snyk](https://snyk.io) as their database is generally more up to date than NPM's